index.bs

```diff
@@ -806,6 +806,12 @@ An attribution source is a [=struct=] with the following items:
 :: A [=trigger-data matching mode=].
 : <dfn>debug cookie set</dfn> (default false)
 :: A [=boolean=].
+: <dfn>attribution scope limit</dfn>
+:: Null or a positive 32-bit integer, number of distinct [=attribution source/attribution scopes=] allowed per [=attribution source/reporting origin=].
+: <dfn>attribution scopes</dfn>
+:: A [=set=] of [=strings=].
+: <dfn>max number of event states</dfn>
+:: Null or a positive 32-bit integer, maximum number of event states that an API caller plans to use.
```

Comment:
`API caller` and `plans` are a bit vague here. Let's either be more specific or defer explanation of this field to elsewhere in the document.

Reply:
Done. referenced existing field.